modules/instrument_builder/jsx/react.elements.js

```diff
@@ -0,0 +1,189 @@
+HeaderElement = React.createClass({
+	render: function(){
+		return (
+			<h2>{this.props.header}</h2>
+		)
+	}
+});
+
+LabelElement = React.createClass({
+	render: function(){
+		return (
+			<p>{this.props.label}</p>
+		)
+	}
+});
+
+ScoredElement = React.createClass({
+	render: function(){
+		return (
+			<div>
+			<label className="lab col-sm-4 col-xs-12">
+				{this.props.label}
+			</label>
+			<div className="col-sm-8">
+				<div className="col-xs-12 element">
+					0
```

Comment:
shouldn't this have a value property instead of all scored elements having a value of 0?